content/en/blog/news/2022-GSoC-2022-Supply-Chain-Security-Final-Report.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "GSoC 2022 Final Report: Improving Supply Chain Security"
+date: 2022-11-08
+draft: false
+weight: 100
+description: >
+  Going through my GSoC journey with Jenkins-X, this works as a documentation for our work to improve supply chain security of Jenkins X
+categories: [blog]
+keywords: [Community, GSoC, 2022, supply chain security, Report]
+slug: "gsoc-2022"
+aliases: []
+author: Osama Magdy
+---
+## Project Description
+
+  Due to the massive increase in open source projects usage in different applications and scales, it became a must to secure every project's supply chain of used packages and dependencies. Supply Chain Security is a rising concern in the current era and it's still under research and investigations. As part of making Jenkins-X an end-to-end solution for CI/CD I worked on both securing our own components and enabling our users to use these features during using our CI/CD for build and release steps.
```

Comment:
Perhaps shift focus to the security threats and that Jenkins X wants to show best practice implementation of Software Supply Chain security. 
Due to massive increase in Attacks on software supply chain, Jenkins X has to make efforts to ensure that the build process is secure. 
https://www.helpnetsecurity.com/2022/02/18/open-source-supply-chain/

Reply:
Done